textalyze.js

```diff
@@ -70,11 +70,19 @@ function analyzeFile(path) {
 
     console.log(`The analysis of the file at ${path} is...`);
     console.log(getPrintStatistics(itemCounts(getChars(sanitizedText))));
-  });  
+  });
 }
 
 if (require.main == module) {
-  analyzeFile('sample_data/moby-dick.full.txt')
+  let textFiles = process.argv.slice(2);
+
+  if (textFiles.length == 0) {
```

Comment:
Since this is the top-level "user interface", I wouldn't throw an error and dump a stack trace here.

Instead, I'd use `console.error(...)` to print an error message to the screen and then call `process.exit(1)` to exit with an error status.